roles/rhoai/defaults/main.yml

```diff
@@ -6,4 +6,13 @@ rhoai_source_namespace: openshift-marketplace
 rhoai_create_dsc: true
 rhoai_dsc_name: default-dsc
 rhoai_wait_for_dsc: true
+rhoai_part_of: rhoai
+rhoai_codeflare: Removed
```

Comment:
all of these variables are part of the DSC cluster but naming doesn't make that clear. I feel this would be better as a map e.g.
```yaml
rhoah_dsc_spec_components:
  codeflare: Removed
  kserve: Managed
  ray: Removed
... etc ..
```

then combine it with whatever is supplied by the user, similar to how `rhoai_operator_map` is built

Reply:
done